ui_testing/testcases/basic_tests/test05_testplans.py

```diff
@@ -22,6 +22,8 @@ def test001_test_plan_delete_testunit(self):
         '''
 
         completed_test_plans = self.test_plan_api.get_completed_testplans()
+        self.base_selenium.LOGGER.info(completed_test_plans)
```

Comment:
You don't need to log all the completed testplans.

Reply:
yes, I added it for tracing and forgot to remove it